projects/dashboards-ng/module-federation/index.ts

```diff
@@ -16,8 +16,8 @@ const setupRemoteComponent = <T>(
 ): Observable<ComponentRef<T>> => {
   const result = new Subject<ComponentRef<T>>();
 
-  loadRemoteModule(factory).then(
-    (module: any) => {
+  loadRemoteModule<Type<T>[]>(factory).then(
```

Comment:
I am unsure, why is this a Type. A type is a component, right? This is the module. Or is a module also a Type.